core/trino-parser/src/main/java/io/trino/sql/ExpressionFormatter.java

```diff
@@ -363,7 +363,7 @@ protected String visitExists(ExistsPredicate node, Void context)
         @Override
         protected String visitIdentifier(Identifier node, Void context)
         {
-            if (!node.isDelimited()) {
+            if (!node.isDelimited() && !ReservedIdentifiers.reservedIdentifiers().contains(node.getValue().toUpperCase())) {
```

Comment:
`reservedIdentifiers` is expensive to call

Reply:
Yeah I realized. Just want to get an opinion if this is the right approach first.
To optimize, should we make `ReservedIdentifiers.sqlKeywords` & `reservedIdentifiers` return static instance instead?

Reply:
Added static members for these 2 methods.